src/views/BridgeTransaction.vue

```diff
@@ -359,6 +368,41 @@ export default class BridgeTransaction extends Mixins(TranslationMixin, LoadingM
     return this.formatCodecNumber(this.soraNetworkFee)
   }
 
+  get isInsufficientBalance (): boolean {
+    if (this.asset) {
+      const fpAmount = this.getFPNumber(this.amount, this.asset.decimals)
+
+      let fpBalance = this.getFPNumberFromCodec(this.asset[this.isSoraToEthereum ? 'balance' : 'externalBalance'], this.asset.decimals)
+
+      if (isXorAccountAsset(this.asset) && this.isSoraToEthereum) {
+        const fpFee = this.getFPNumberFromCodec(this.soraNetworkFee, this.asset.decimals)
+        fpBalance = fpBalance.sub(fpFee)
+      }
+
+      if (FPNumber.lt(fpBalance, fpAmount)) {
+        this.insufficientBalanceAssetSymbol = this.asset ? formatAssetSymbol(this.asset.symbol, !this.isSoraToEthereum) : ''
+        return true
+      }
+    }
+    return false
+  }
+
+  get isInsufficientXorForFee (): boolean {
```

Comment:
`isInsufficientXorForFee ` `isInsufficientEthereumForFee ` are the same as in the `Bridge.vue`, could we reuse them by creating mixin or utility functions?